### ISSUE TITLE
Product detail dialog has no obvious close and weird pattern

### ISSUE THREAD

**Issue by timheuer** [Microsoft/InventorySample#19]
It seems weird that the product details is a popup secondary window where the other ones like order details, etc. are not.  It is a different pattern than the rest of the app.

Additionally, the window opened doesn't have an obvious close method other than the Red X

**Comment by marb2000** [comment]
Agree. The current Master/Details Product view only has the Master, we should 1) implement the details part, 2) make possible the user can  switch from Griid view (the current one) to a List view, 3) allow the user to see the details in a different view when double click (or using the Open in a View button).


About Closing the view. We are following the Mail app pattern where there is no close button, the only way to close the view is using the X on the window. Thank yyou for your feedback, I will discuss with our design team.
